google-cloud-pubsub/lib/google/cloud/pubsub/subscription.rb

```diff
@@ -256,6 +256,47 @@ def labels= new_labels
           @resource_name = nil
         end
 
+        ##
+        # The duration (in seconds) for when a subscription expires after the
+        # subscription goes inactive. A subscription is considered active as
+        # long as any connected subscriber is successfully consuming messages
+        # from the subscription or is issuing operations on the subscription.
+        #
+        # If `expires_in` is not set, a *default* value of of 31 days will be
+        # used. The minimum allowed value is 1 day.
+        #
+        # Makes an API call to retrieve the labels value when called on a
+        # reference object. See {#reference?}.
+        #
+        # @return [Numeric, nil] The expiration duration, or `nil` if unset.
+        #
+        def expires_in
+          ensure_grpc!
+
+          return nil if @grpc.expiration_policy.nil?
+
+          Convert.duration_to_number @grpc.expiration_policy.ttl
+        end
+
+        ##
+        # Sets the duration (in seconds) for when a subscription expires after
+        # the subscription goes inactive.
+        #
+        # @param [Numeric, nil] ttl The expiration duration, or `nil` to unset.
```

Comment:
what is the unit? days? seconds? millis?

Reply:
Ah, I see now `seconds` in the description, sorry. But wouldn't hurt to repeat it.

Reply:
Added.